stream-utils.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
```

Comment:
This file must be added to https://github.com/sindresorhus/globby/blob/04d51bf1b85ad9c7ec968126b26c6d6b7eae92be/package.json#L21